.github/workflows/main.yml

```
@@ -0,0 +1,55 @@
+name: CI
+
+on:
+  push:
+    branches: ['master']
+  pull_request:
+    branches: ['*']
+
+jobs:
+  build_and_test:
+    name: LNDK Rust Build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Looks like we can also make this v3

```suggestion
      - uses: actions/checkout@v3
```